platform/engineering/scheduled-downtime-process.md

```diff
@@ -0,0 +1,10 @@
+# Scheduled Downtime Process for VA.gov
+
+## Overview
+Any site upgrades, migration, or changes that affect will resolute in the public site to experience downtime, should follow the process below in order to allow the right parties to be aware, give enough time for other teams to make any preparations, and to minimize any effect to the Veterans. 
```

Comment:
Proposed edit:

Site upgrades, migrations, or changes that affect the public site may result in the site requiring VA.gov services to be taken down. When this is necessary, we need to make sure that the VA.gov users and developers working on the VA.gov platform are sufficiently informed. Please follow the process below in order to allow the right parties to be aware, give enough time for other teams to make any preparations, and to minimize effect to the Veterans and other users of VA.gov.